Cargo.toml

```
@@ -101,6 +101,7 @@ interprocess = { git = "https://github.com/kotauskas/interprocess", rev = "7fa31
 indoc = "1"
 lazycell = "1"
 lazy_static = "1.4"
+lychee-lib = "0.10"
```

can we try enabling the [vendored-openssl](https://github.com/lycheeverse/lychee/blob/master/lychee-lib/Cargo.toml#L68) feature to see if that fixes our windows woes? this should be enabled regardless, even if it doesn't fix the issue.

```suggestion
lychee-lib = { version = "0.10", features = ["vendored-openssl"] }
```

---

Already tested that here: [`ea9f375` (#1328)](https://github.com/apollographql/rover/pull/1328/commits/ea9f375aff3c49e88d4d2fabde4cb6772a898cf4)
It throws another error about wrong Perl implementation while building

---

well boo! that's unfortunate. i'd still like to enable `vendored-openssl` if possible so contributors don't have to worry about installing or updating it (because it's a massive pain). 

want to try installing strawberry perl to see if that fixes? https://strawberryperl.com/